<launch>
  <include file="$(find spot_move_base)/launch/spot_move_base_sim.launch" />

  <arg name="rosbag_path" default="" />
  <arg name="save_data_path" default="" />

  <param name="rosbag_path" type="str" value="$(arg rosbag_path)" />
  <param name="save_data_path" type="str" value="$(arg save_data_path)" />
  <param name="robot_name" type="str" value="spot" />

  <node name="node_parser" pkg="scand-parser" type="parser.py" output="screen" required="true"/>
</launch>
